README.md

```diff
@@ -24,7 +24,7 @@ Variable    | Description | Examples
 -------------|------------ |---------------
 aws_region  | AWS region to create the infrastructure  | eu-west-2
 environment  | AWS environment in which to create the infrastructure  | development
-prometheus_web_fqdn | Prometheus targets | web.
+prometheus_tag_regex | Prometheus tag | my-instance-tag
```

Comment:
Can we keep these ordered for consistency please? It makes things easier to find too